### PR TITLE
add componengt: Loading

### PR DESCRIPTION
**Pull request by fondadam**
pr

### CODE REVIEW

**Comment by rock-zhang** [comment]
过

**Comment by farzer** [comment]
no p

**Comment by heyong-2015** [comment]
pass

File: doc/Loading.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+## 组件介绍
+
+### Loading
+```javascript
+Loading.propTypes = {
+    style: PropTypes.object,
+    size: PropTypes.number
+};
```
Comment (rock-zhang) [inline doc/Loading.md]: 111

File: doc/Loading.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+```javascript
+Loading.defaultProps = {
```
Comment (rock-zhang) [inline doc/Loading.md]: 222

**Pull request merged**